Classes/Backend/DataHandler/MetaDataHandler.php

```diff
@@ -13,11 +13,11 @@ protected function updateJobData(&$fieldArray)
     {
         $queueDataFactory = new QueueDataFactory();
         $job = new Job();
-        $job->setSerializedData($fieldArray['serialized_data'] ?? '');
+        $job->setSerializedData(json_encode(json_decode($fieldArray['serialized_data'])) ?? '');
```

Comment:
@mkarulin The operation "?? ''" was meant to deal with the index "serialized_data" of the array $fieldArray not being set, which can happen if the bulk edit functionality was used. I think the operation should be moved into the parenthesis instead of having it on the outmost level.

$job->setSerializedData(json_encode(json_decode($fieldArray['serialized_data'] ?? '')));

Or do I misunderstand something at this point?